modules/migration-mtc-release-notes-1-7-04.adoc

```diff
@@ -0,0 +1,20 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7-04_{context}"]
+= {mtc-full} 1.7.4 release notes
+
+[id="resolved-issues-1-7-04_{context}"]
+== Resolved issues
+
+There are no major resolved issues in this release.
+
+
+[id="known-issues-1-7-04_{context}"]
+== Known issues
+
+.Rollback missing out deletion of some resources from the target cluster
```

Comment:
```suggestion
.Rollback missing out deletion of some resources from the target cluster
```
Clarify this title. Do you mean "Rollback is missing the deletion of some resources from the target cluster"?